dobackup.sh

```diff
@@ -15,10 +15,21 @@ else
     echo "TARGET env var is set"
 fi
 
+if [ -z "${S3_ENDPOINT}" ]; then
+  AWS_ARGS=""
+else
+  AWS_ARGS="--endpoint-url ${S3_ENDPOINT}"
+fi
+
 echo "creating archive"
 tar -zcvf $FILE_NAME $TARGET
 echo "uploading archive to S3 [$FILE_NAME, storage class - $S3_STORAGE_CLASS]"
-aws s3 cp --storage-class $S3_STORAGE_CLASS $FILE_NAME $S3_BUCKET_URL
+aws s3 $AWS_ARGS cp --storage-class $S3_STORAGE_CLASS $FILE_NAME $S3_BUCKET_URL
 echo "removing local archive"
 rm $FILE_NAME
 echo "done"
+
+if [ -n $HEALTHCHECKS_IO_CHECK_ID ]; then
+    echo "Notifying healthchecks.io"
+    curl -m 10 --retry 5 https://hc-ping.com/$HEALTHCHECKS_IO_CHECK_ID
```

Comment:
healthchecks.io seems like a useful service. I am fine with adding this! I know statuscake.com offers a similar service (https://www.statuscake.com/kb/knowledge-base/what-is-push-monitoring/) so it might be nice to check if a `$WEBHOOK_URL` is defined, and if so try to `GET` it using curl. That would make this generic up to the level that you could use it with whatever service that supports `GET` requests (maybe even some internal tooling in your organisation).

Do you feel like doing that small rewrite?